packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Models/Types/MrwSerializationTypeProvider.cs

```diff
@@ -51,6 +52,8 @@ public MrwSerializationTypeProvider(ModelProvider model, InputModelType inputMod
             Namespace = model.Namespace;
         }
 
+        protected override string GetFilename() => Path.Combine("src", "Generated", "Models", $"{Name}.serialization.cs");
```

Comment:
Looks like the filename is lowercase here.

Reply:
It would be good to run the Unbranded-Typespec launch profile locally to make sure there are no diffs.

Reply:
I will fix this

Reply:
that is part of CI to run that and check for diffs
![image](https://github.com/microsoft/typespec/assets/64171366/60651103-ed0c-4ac6-874b-5ae0ee313e42)